Build fails on NixOS

```
[1/5] Compiling C++ object src/libhyprgrass.so.p/GestureManager.cpp.o
FAILED: src/libhyprgrass.so.p/GestureManager.cpp.o
g++ -Isrc/libhyprgrass.so.p -Isrc -I../src -I/nix/store/518ba5vj332la1af6nyc12j8ash59rwl-wf-touch-git-dev/include -I/nix/store/mhrvlcdplm5rnw7554pqxg05nxakgm46-glm-0.9.9.8/include -I/nix/store/il59v28gchkxs2xhn1llin96ci60y1zh-pixman-0.42.2/include/pixman-1 -I/nix/store/vlkhrqkf7scs3rr260w5cj58csn5vvf8-libdrm-2.4.115-dev/include -I/nix/store/vlkhrqkf7scs3rr260w5cj58csn5vvf8-libdrm-2.4.115-dev/include/libdrm -I/nix/store/zns0kczk280c3hni52aiz6vpf7vj354d-hyprland-0.29.1+date=2023-09-09_1a6f961-dev/include -I/nix/store/zns0kczk280c3hni52aiz6vpf7vj354d-hyprland-0.29.1+date=2023-09-09_1a6f961-dev/include/hyprland/protocols -I/nix/store/zns0kczk280c3hni52aiz6vpf7vj354d-hyprland-0.29.1+date=2023-09-09_1a6f961-dev/include/hyprland/wlroots -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -std=c++23 -fPIC -DWLR_USE_UNSTABLE -MD -MQ src/libhyprgrass.so.p/GestureManager.cpp.o -MF src/libhyprgrass.so.p/GestureManager.cpp.o.d -o src/libhyprgrass.so.p/GestureManager.cpp.o -c ../src/GestureManager.cpp
In file included from ../src/GestureManager.hpp:4,
                 from ../src/GestureManager.cpp:1:
/nix/store/zns0kczk280c3hni52aiz6vpf7vj354d-hyprland-0.29.1+date=2023-09-09_1a6f961-dev/include/hyprland/src/debug/Log.hpp:4:10: fatal error: format: No such file or directory
    4 | #include <format>
      |          ^~~~~~~~
compilation terminated.
[2/5] Compiling C++ object src/libhyprgrass.so.p/main.cpp.o
FAILED: src/libhyprgrass.so.p/main.cpp.o
g++ -Isrc/libhyprgrass.so.p -Isrc -I../src -I/nix/store/518ba5vj332la1af6nyc12j8ash59rwl-wf-touch-git-dev/include -I/nix/store/mhrvlcdplm5rnw7554pqxg05nxakgm46-glm-0.9.9.8/include -I/nix/store/il59v28gchkxs2xhn1llin96ci60y1zh-pixman-0.42.2/include/pixman-1 -I/nix/store/vlkhrqkf7scs3rr260w5cj58csn5vvf8-libdrm-2.4.115-dev/include -I/nix/store/vlkhrqkf7scs3rr260w5cj58csn5vvf8-libdrm-2.4.115-dev/include/libdrm -I/nix/store/zns0kczk280c3hni52aiz6vpf7vj354d-hyprland-0.29.1+date=2023-09-09_1a6f961-dev/include -I/nix/store/zns0kczk280c3hni52aiz6vpf7vj354d-hyprland-0.29.1+date=2023-09-09_1a6f961-dev/include/hyprland/protocols -I/nix/store/zns0kczk280c3hni52aiz6vpf7vj354d-hyprland-0.29.1+date=2023-09-09_1a6f961-dev/include/hyprland/wlroots -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -std=c++23 -fPIC -DWLR_USE_UNSTABLE -MD -MQ src/libhyprgrass.so.p/main.cpp.o -MF src/libhyprgrass.so.p/main.cpp.o.d -o src/libhyprgrass.so.p/main.cpp.o -c ../src/main.cpp
In file included from ../src/GestureManager.hpp:4,
                 from ../src/main.cpp:1:
/nix/store/zns0kczk280c3hni52aiz6vpf7vj354d-hyprland-0.29.1+date=2023-09-09_1a6f961-dev/include/hyprland/src/debug/Log.hpp:4:10: fatal error: format: No such file or directory
    4 | #include <format>
      |          ^~~~~~~~
compilation terminated.
[ 90%] Built target nvim
[ 90%] Generating doc/tags
[ 90%] Generating pack/dist/opt/matchit/doc/tags
[ 90%] Generating nvim.pot
[3/5] Compiling C++ object src/gestures/libgestures.a.p/Gestures.cpp.o
ninja: build stopped: subcommand failed.
error: builder for '/nix/store/1bgxk57dc5dxv869hlzjmi7k8y2g99k4-hyprgrass-0.3.0.drv' failed with exit code 1;
       last 10 log lines:
       > FAILED: src/libhyprgrass.so.p/main.cpp.o
       > g++ -Isrc/libhyprgrass.so.p -Isrc -I../src -I/nix/store/518ba5vj332la1af6nyc12j8ash59rwl-wf-touch-git-dev/include -I/nix/store/mhrvlcdplm5rnw7554pqxg05nxakgm46-glm-0.9.9.8/include -I/nix/store/il59v28gchkxs2xhn1llin96ci60y1zh-pixman-0.42.2/include/pixman-1 -I/nix/store/vlkhrqkf7scs3rr260w5cj58csn5vvf8-libdrm-2.4.115-dev/include -I/nix/store/vlkhrqkf7scs3rr260w5cj58csn5vvf8-libdrm-2.4.115-dev/include/libdrm -I/nix/store/zns0kczk280c3hni52aiz6vpf7vj354d-hyprland-0.29.1+date=2023-09-09_1a6f961-dev/include -I/nix/store/zns0kczk280c3hni52aiz6vpf7vj354d-hyprland-0.29.1+date=2023-09-09_1a6f961-dev/include/hyprland/protocols -I/nix/store/zns0kczk280c3hni52aiz6vpf7vj354d-hyprland-0.29.1+date=2023-09-09_1a6f961-dev/include/hyprland/wlroots -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -std=c++23 -fPIC -DWLR_USE_UNSTABLE -MD -MQ src/libhyprgrass.so.p/main.cpp.o -MF src/libhyprgrass.so.p/main.cpp.o.d -o src/libhyprgrass.so.p/main.cpp.o -c ../src/main.cpp
       > In file included from ../src/GestureManager.hpp:4,
       >                  from ../src/main.cpp:1:
       > /nix/store/zns0kczk280c3hni52aiz6vpf7vj354d-hyprland-0.29.1+date=2023-09-09_1a6f961-dev/include/hyprland/src/debug/Log.hpp:4:10: fatal error: format: No such file or directory
       >     4 | #include <format>
       >       |          ^~~~~~~~
       > compilation terminated.
       > [3/5] Compiling C++ object src/gestures/libgestures.a.p/Gestures.cpp.o
       > ninja: build stopped: subcommand failed.
       For full logs, run 'nix log /nix/store/1bgxk57dc5dxv869hlzjmi7k8y2g99k4-hyprgrass-0.3.0.drv'.
error: 1 dependencies of derivation '/nix/store/rhdx8sgiy1z39266ihmsinqndb6pd77k-hm_hyprhyprland.conf.drv' failed to build
error: 1 dependencies of derivation '/nix/store/dwcs4q0d7jy9jrgkhsy8bb3jad1gikx7-activation-script.drv' failed to build
error: 1 dependencies of derivation '/nix/store/kia82i1igaqqkii0mm7hwanq6gp4bpbw-home-manager-generation.drv' failed to build
error: 1 dependencies of derivation '/nix/store/asbmdjx4vnfyx16mizwk6l6pfvflphn0-unit-home-manager-matt.service.drv' failed to build
error: 1 dependencies of derivation '/nix/store/j7wasq6a852jnpsbqmwmiljhhdg0xlsl-system-units.drv' failed to build
error: 1 dependencies of derivation '/nix/store/iy68g5lv0nff6zd4n2fmdn5ximxmsi48-etc.drv' failed to build
error: 1 dependencies of derivation '/nix/store/45ml2asrfdyi5j8csp5wzr41c0a81k8v-nixos-system-wim-23.11pre519881.3efb0f6f404e.drv' failed to build
```

--- a/nix/default.nix
+++ b/nix/default.nix
@@ -1,6 +1,6 @@
 {
   lib,
-  stdenv,
+  gcc13Stdenv,
   cmake,
   hyprland,
   wf-touch,
@@ -8,7 +8,7 @@
 }: let
   pluginInfo = builtins.fromTOML (builtins.readFile ../hyprload.toml);
 in
-  stdenv.mkDerivation {
+  gcc13Stdenv.mkDerivation {
     pname = "hyprgrass";
     version = pluginInfo.hyprgrass.version;
     src = ./..;

